### DISCUSSION TITLE
Approach when not using `result` in the useMutation tuple?

### DISCUSSION THREAD

**Discussion by GollyJer** [urql-graphql/urql#1212] [Q&A] [answered]
Hi! We're switching over from Apollo so I'll have a few questions as we go. 😀
This is a simple one. More of a curiosity.

We use `optimistic` updates for all mutations. With useMutation returning a tuple this leaves us with a ton of linter warnings because we don't need the result.

<img width=550 src=https://user-images.githubusercontent.com/689204/101722574-5fe3c980-3a5f-11eb-933c-915f91c16285.png />

As a fix we're using 
<img width=650 src=https://user-images.githubusercontent.com/689204/101722732-b81acb80-3a5f-11eb-9c15-4c08c8080e37.png />
Which I don't love.

Is there a better approach?

**Comment by kitten** [comment, marked as answer]
Destructuring an array doesn't require you to choose a name, so you can use: `const [, update] = useMutation(...);` for instance.

It's also possible to set up an exception for ESLint and TypeScript to ignore variables that start with an underscore, e.g. `const [_result, update] = useMutation(...);`

**Reply by GollyJer** [reply]
> Destructuring an array doesn't require you to choose a name...

With years of working in React it's comical I didn't know this. 😂
Goes to show there's never a stupid question. 😃

Thanks Phil.
